src/Modal.js

```diff
@@ -156,6 +160,19 @@ const Modal = React.createClass({
     if (this.props.keyboard && e.keyCode === 27) {
       this.props.onRequestHide();
     }
+  },
+
+  focusModalContent () {
+    this.lastFocus = document.activeElement;
```

Comment:
@taion Good catch, though since this PR has already been merged lets create an issue (#729) for this and follow up on another PR.